### ISSUE TITLE
Console error when opening character sheets

### ISSUE THREAD

**Issue by tposney** [tposney/midi-qol#987]
In GitLab by @michael.j.tait98 on Sep 5, 2022, 02:26

A red console error is displayed when opening some character sheets.
![image](https://my-gitlab-bucket.s3.amazonaws.com/640711674/66e33b54c9f0ac9a66a4c8a856be4ab5176f46d84d6120c0737092dc13a698a7/image.png)

Noticed on Foundry - V10, Build 284.

5e Version - 2.0.2.

Midi QOL version - 10.0.7.

Tidy5eSheet - 0.6.4.

Triggers on character sheet open, either from canvas or actor tab. Attached an image of error, an actor the problem appears on, and Midi workflow settings.

[fvtt-Actor-kavaxokhar-dorun.json](https://my-gitlab-bucket.s3.amazonaws.com/640711674/1c06f12a683da4e9b2c403fdbbd8e12255a0f2ede9243c1df57451b6508f7743/fvtt-Actor-kavaxokhar-dorun.json)
[fvtt-midi-qol-settings.json](https://my-gitlab-bucket.s3.amazonaws.com/640711674/bb80c15e157f0f5193ff6039cfe4e65510ca9b441ea3c0655b47e21d502ba6c0/fvtt-midi-qol-settings.json)

**Comment by tposney** [comment]
I tried to reproduce the error but could not. Having said that I did find 2 problems in the code that produce errors, just not the one you are seeing.

It could be a timing issue as one of the issues is that dnd5e getChatData became an async call which was not being properly processed and that might end up resolving your issue as well.

Changes will go out in 10.0.8

**Comment by tposney** [comment]
If that does not resolve the issue please reopen.

**Comment by tposney** [comment]
In GitLab by @michael.j.tait98 on Sep 6, 2022, 06:59

From what I can tell after updating a moment ago, the issue still exists for me. I've attached a full image of the error incase there is more info there, I noticed it mention MATT for some reason. I've also attached another character that oddly doesn't give me the error, despite the other 3 doing so.

![image](https://my-gitlab-bucket.s3.amazonaws.com/640711674/0b9c1443ba88466a4994baa259dbe5b4793015211eaacfa5ccfe6c25654c21ca/image.png)
